lab-trent/karma.conf.js

```diff
@@ -0,0 +1,26 @@
+const webpackConfig = require('./webpack.config.js');
+webpackConfig.entry = { };
```

Comment:
This empty object will break your tests with Webpack 2.x. You need to delete the entry prop.